apps/docs/src/app/core/component-docs/popover/examples/popover-simple/popover-example.component.html

```diff
@@ -1,113 +1,113 @@
-                    </div>
-                </div>
-            </div>
-            <ul fd-list>
-                <li fd-list-item *ngFor="let option of list1">
-                    <a [attr.href]="option.url" fd-list-title>{{option.text}}</a>
-                </li>
-            </ul>
-        </fd-popover-body>
-    </fd-popover>
-
-    <fd-popover>
-        <fd-popover-control>
-            <button fd-button label="Button"></button>
-        </fd-popover-control>
-        <fd-popover-body *ngIf="list1 && list1.length">
-            <div fd-popover-body-header>
-                <div fd-bar [barDesign]="'header-with-subheader'" [cozy]="true">
-                    <div fd-bar-left>
-                        <fd-bar-element>Cozy Header</fd-bar-element>
-                    </div>
-                </div>
-                <div fd-bar [barDesign]="'subheader'" [cozy]="true">
-                    <div fd-bar-middle>
-                        <fd-bar-element [fullWidth]="true"
-                            >SubHeader Middle section with Full Width Element
-                        </fd-bar-element>
-                    </div>
-                </div>
-            </div>
-            <ul fd-list>
-                <li fd-list-item *ngFor="let option of list1">
-                    <a [attr.href]="option.url" fd-list-title>{{option.text}}</a>
-                </li>
-            </ul>
-            <div fd-popover-body-footer>
-                <div fd-bar [barDesign]="'footer'" [cozy]="true">
-                    <div fd-bar-right>
-                        <fd-button-bar label="Save" fdType="emphasized"></fd-button-bar>
-                        <fd-button-bar label="Cancel"></fd-button-bar>
-                    </div>
-                </div>
-            </div>
-        </fd-popover-body>
-    </fd-popover>
-
-    <fd-popover placement="bottom">
-        <fd-popover-control>
-            <fd-avatar size="s" [circle]="true" image="https://picsum.photos/id/2/400/400"></fd-avatar>
-        </fd-popover-control>
-        <fd-popover-body *ngIf="list1 && list1.length">
-            <ul fd-list>
-                <li fd-list-item *ngFor="let option of list1">
-                    <a [attr.href]="option.url" fd-list-title>{{option.text}}</a>
-                </li>
-            </ul>
-        </fd-popover-body>
-    </fd-popover>
-
-    <fd-popover
-        placement="bottom"
-        [fixedPosition]="true"
-        [noArrow]="false"
-    >
-        <fd-popover-control>
-            <button fd-button label="Fixed Position"></button>
-        </fd-popover-control>
-        <fd-popover-body *ngIf="list1 && list1.length">
-            <ul fd-list>
-                <li fd-list-item *ngFor="let option of list1">
-                    <a [attr.href]="option.url" fd-list-title>{{option.text}}</a>
-                </li>
-            </ul>
-        </fd-popover-body>
-    </fd-popover>
-</div>
+<div class="fd-docs-flex-display-helper">
```

Comment:
please configure your vscode to avoid formatting changes. It's hard to see now where the changes are